references/defer-api/get-execution.mdx

```diff
@@ -55,7 +55,7 @@ GET https://api.defer.run/public/v1/executions/{execution-id}
 
 <ResponseField
   name="state"
-  type="created | running | succeed | failed | aborting | aborted | cancelled"
+  type="created | running | succeed | failed | aborting | aborted | cancelled | cancelling | started"
```

Comment:
```suggestion
  type="created | started | succeed | failed | cancelled | aborting | aborted | discarded"
```
I have submitted a patch to correct the client types as follows.

Reply:
perfect! should i close this, then?